domain/schema/model/sql/0011-storage.sql

```diff
@@ -108,10 +108,31 @@ CREATE TABLE unit_storage_directive (
 CREATE INDEX idx_unit_storage_directive
 ON unit_storage_directive (unit_uuid);
 
+-- storage_kind defines what type Juju considers a storage instance in the model
+-- to be of. While we have the concept of charm storage kind it is not
+-- necessarily the same as the storage instance. This is even more true when we
+-- are trying to understand the composition of a storage_instance and not the
+-- purpose it may be fulfilling.
+--
+-- It is possible for a storage_instance to not be associated to a charm in the
```

Comment:
```sql
-- It is possible for a storage_instance to not be associated to a charm in the
-- model.
```

This comment feels disjointed, perhaps it should be on the `storage_instance` table's comment?